test/fixtures/xruby/lib/alpha_cases.rb

```diff
@@ -0,0 +1,20 @@
+class AlphaCase < OpenStruct
```

Comment:
Although that is also the name of a computer hardware manufacturer..
Naming is hard.